.github/workflows/build-package.yml

```diff
@@ -0,0 +1,52 @@
+name: Build Package
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PACKAGING_PASSPHRASE: ${{ secrets.PACKAGING_PASSPHRASE }}
+  PACKAGING_SECRET_KEY: ${{ secrets.PACKAGING_SECRET_KEY }}
+
+on:
+  push:
+    branches: "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ el/7, el/8, ol/7, debian/stretch, debian/buster, ubuntu/xenial, ubuntu/bionic,ubuntu/focal ]
+#        platform: [ debian/stretch ]
+    env:
+      PLATFORM: ${{ matrix.platform }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      #      - name: install dependencies
+      #        run: sudo apt install libcurl4-openssl-dev libssl-dev
+      #
+      #      - name: Define git credentails
+      #        run: git config --global user.email "gindibay@microsoft.com"&& git config --global user.name "Gurkan Indibay"
+
+      - name: Clone tools branch
+        run: git clone --depth=1 --branch tools-transition-update-docker https://github.com/citusdata/tools.git tools
```

Comment:
It would be nicer to create a new tag on the tools repo, and checkout the tag. We can clone develop for now, but we need to start using tags at some poing

Reply:
Created v0.8.0 tag and used this tag in pipelines as tools source